.goreleaser.yml

```diff
@@ -12,7 +12,7 @@ builds:
   - arm
   - arm64
   ldflags:
-  - -X main.Version={{.Version}}
+  - -X main.version={{.Version}}
```

Comment:
This fix is not correct, the variable holding the version is called `Version` in package main, see: https://github.com/matryer/moq/blob/main/main.go#L17

Reply:
Thanks a million for the explanation! Closing this due to my misunderstanding